lab-abigail/test/node-test.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const ll = require('../lib/linked-list');
+const Node = require('../lib/list-node');
+const chai = require('chai');
+const expect = chai.expect;
+
+describe('Node Tests' , function() {
+
+  describe('Reverse Prototype', function() {
+
+    let list = new Node();
+    list.append(1);
```

Comment:
**TODO FOR LOST POINTS**

I got an error when I ran your tests that ``list.append is not a function``. I think it's because you meant to instantiate your ``ll` constructor at line 3, but you're actually trying to append to your Node constructor instead because ``list`` is an instance of Node.